.github/workflows/ci.yml

```diff
@@ -247,16 +247,15 @@ jobs:
         run: yarn run build
 
       - name: pack
-        # Cannot use yarn lerna run pack
-        run: yarn lerna exec yarn pack
+        run: yarn workspaces foreach -A exec yarn pack
 
       # Prepack (without cleanup per package) to ensure that type resolution in
       # dependent packages continues to work when the typedefs are generated by
       # their upstream packages. This helps avoid a situation in which the types
       # only resolve because of the state of the local filesystem, and fails
       # when imported in an NPM node_modules tree.
       - name: Prepack packages
-        run: yarn lerna run --reject-cycles --concurrency 1 prepack
+        run: yarn workspaces foreach -A run prepack
```

Comment:
please leave this one. this change. loses cycle detection.

it also loses topological ordering, though `--topological` would restore that.